sdk-network-providers/transactions.md

```diff
@@ -0,0 +1,36 @@
+class TransactionOnNetwork:
+    is_completed: Optional[boolean]
+    hash: str
+    type: str
+    nonce: int
+    round: int
+    epoch: int
+    value: int
+    receiver: IAddress
+    sender: IAddress
+    gas_limit: int
+    gas_price: int
+    data: str
+    signature: str
+    status: TransactionStatus
```

Comment:
I would keep this out of the transaction definition. Reason: the processes involved in deciding the status of the transaction are pretty hard on the proxy/nodes

Reply:
This parameter comes from all APIs and is needed if you want to resync until you get status != pending